add ignoreRoutes for SentryNavigatorObserver in sentry-dart (#11018)
getsentry/sentry-docs: docs/platforms/flutter/integrations/routing-instrumentation.mdx

)
```

### Ignore Routes

> Set `ignoreRoutes` if you want routes to be ignored and not pushed to Sentry servers.

Reviewer: I think `not pushed to Sentry servers` is a bit generic.

```suggestion
Set `ignoreRoutes` if you want routes to be ignored and not processed by the navigation observer.
```
